hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/filtering/chain/FilterChain.java

```diff
@@ -0,0 +1,28 @@
+package pl.allegro.tech.hermes.consumers.consumer.filtering.chain;
+
+import pl.allegro.tech.hermes.consumers.consumer.Message;
+import pl.allegro.tech.hermes.consumers.consumer.filtering.MessageFilter;
+
+import java.util.List;
+
+public class FilterChain {
+
+    public FilterChain(final List<MessageFilter> messageFilters) {
+        this.messageFilters = messageFilters;
+    }
+
+    private List<MessageFilter> messageFilters;
```

Comment:
All immutable classes in hermes have got fields declared at the class beginning with a final word. Imho it would be nice to keep that consistency.